packages/web-components/stencil.config.ts

```diff
@@ -54,4 +55,9 @@ export const config: Config = {
       serviceWorker: null, // disable service workers
     },
   ],
+  testing: {
+    moduleNameMapper: {
+      '^.+.(svg)$': 'jest-transformer-svg',
```

Comment:
Do you mind mentioning why we need this or what it does?

Reply:
This is so the jest spec test can import the svg without error. We haven't needed this elsewhere because we've been using e2e tests for almost all tests. The spec tests run faster though. [Docs](https://stenciljs.com/docs/unit-testing)